include/devicetree/mbox.h

```diff
@@ -0,0 +1,100 @@
+/**
+ * @file
+ * @brief MBOX Devicetree macro public API header file.
+ */
+
+/*
+ * Copyright (c) 2022 Carlo Caione <ccaione@baylibre.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DEVICETREE_MBOX_H_
+#define ZEPHYR_INCLUDE_DEVICETREE_MBOX_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup devicetree-mbox Devicetree MBOX API
```

Comment:
Thanks! Please add documentation for this new group in the devicetree API reference .rst page  too. Grep for the other groups to see where this should go. otherwise LGTM.

Reply:
done